plugins/LanguagesManager/LanguagesManager.php

```diff
@@ -137,18 +128,46 @@ public function uninstall()
     }
 
     /**
+     * First check forced by request language setting,
+     * then - session language setting, then - from browser,
+     * and finally default configured setting.
+     *
      * @return string Two letters language code, eg. "fr"
      */
-    public static function getLanguageCodeForCurrentUser()
+    public static function getCurrentLanguage()
```

Comment:
This method could be private as the Translator's method should be used instead? (i.e. nobody should use this method)